server/test/lib/scheduler/Schedule.spec.ts

```diff
@@ -100,38 +101,105 @@ describe("Schedule class", function() {
     });
   });
 
+  describe("getTreesWorkedOnLeastRecently", function() {
+    const rootWorkspaces = [
+      workspaces.get("1"),
+      workspaces.get("2"),
+      workspaces.get("3"),
+      workspaces.get("4"),
+      workspaces.get("5"),
+    ];
+
+    it("works in straightforward case", async function() {
```

Comment:
"in a straightforward case" seems like a context.

A more elegant way of doing this would be to have these as contexts, and then have tests for each expect block. ("has the correct length")